tools/pre-commit.sh

```diff
@@ -23,6 +25,9 @@ then
     exit $need_format
 fi
 
+#TODO Do we need the `if` cluases above for these below?
+echo "Performing checks with yamllint"
 yamllint --no-warnings ./
```

Comment:
If yamllint has a --check mode similar to black, then I think that makes.

Flake does not, as it doesn't actually auto-format things - it just returns with errors if the checks fail.